packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1386,25 +1433,65 @@ export const components: ThemeOptions["components"] = {
   MuiRadio: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <></>,
```

Comment:
Are there supposed to be icons here or are we using a circle?

Does this accept `null`? I personally prefer explicitly defining `<Fragment>` if we could, but `null` should be valid for a "null component".

Reply:
It threw an error when I tried `null`

Reply:
Yep! We verified this behavior. Fragments are fine.